devel/nodejs15/Portfile

```diff
@@ -9,9 +9,6 @@ PortGroup               deprecated 1.0
 # https://github.com/nodejs/Release
 deprecated.eol_version  yes
 
-# on macOS nodejs only builds against libc++
-# this force is OK as node does not link against any other c++ libs
-depends_lib-append      port:libcxx
 configure.cxx_stdlib    libc++
```

Comment:
This should actually be removed, it is both wrong and redundant. MacPorts already uses `libc++` on 10.6+ by default, no point in forcing that. And why nodeJS does not build with `libstdc++` is a silly small bug in its code where a macro is wrong. Solution is to fix the bug, not prohibit using the runtime.